File: Node/src/dto/enums/NoticeCategory.ts
/// Описывает категорию уведомлений
/// - `APPOINTMENT_EMPLOYEE`: Уведомления о рабочих аппоинтментах
/// - `APPOINTMENT_CUSTOMER`: Уведомления о кастомерских аппоинтментах
/// - `EMPLOYEE`            : Уведомления рабочему (мб оверхед)
/// - `POSITION`            : Уведомления о изменении позиции
/// - `SALARY`              : Уведомления о зарплате
/// - `SUGGEST`             : Предложение о смерживании контактов
export enum NoticeCategory {
    appointmentEmployee = "appointmentEmployee",
    appointmentCustomer = "appointmentCustomer",
    employee = "employee",
    position = "position",
    salary = "salary",
    suggest = "suggest",
}

File: Android/shared/src/main/kotlin/shared/dto/enums/NoticeCategory.kt
package shared.dto.enums

import kotlinx.serialization.SerialName

/// Описывает категорию уведомлений
/// - `APPOINTMENT`: Уведомления о рабочих аппоинтментах
/// - `ASSIGNMENT`: Уведомления о кастомерских аппоинтментах
/// - `EMPLOYEE`            : Уведомления рабочему (мб оверхед)
/// - `POSITION`            : Уведомления о изменении позиции
/// - `SALARY`              : Уведомления о зарплате
/// - `SUGGEST`             : Предложение о смерживании контактов
enum class NoticeCategory {
    @SerialName("appointment")
    APPOINTMENT,

    @SerialName("assignment")
    ASSIGNMENT,

    @SerialName("employee")
    EMPLOYEE,

    @SerialName("position")
    POSITION,

    @SerialName("salary")
    SALARY,

    @SerialName("suggest")
    SUGGEST,
}
